core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-or-update-solution-modal.component.ts

```diff
@@ -140,8 +141,12 @@ export class AddOrUpdateSolutionModalComponent
 
   ngOnInit(): void {
     this.solutionType = this.stateSolutionService.savedMemento;
-    this.savedMemento = (
-      this.stateSolutionService.savedMemento?.correctAnswer);
+    if (this.solutionType) {
+      this.savedMemento = (
+        this.solutionType?.correctAnswer);
```

Comment:
Why `?`, we prefer not to use it right?

Reply:
Done, no need, changes made by some other contributor.